docs/MAINTAINER_VISION.md

```
@@ -0,0 +1,170 @@
+
+### Files
+- `crates/<crate>/src/lib.rs` - <change>
+- `specs/features/<area>/<feature>.feature` - <scenarios>
+- `fixtures/...` - <test data>
+
+### Verification
+```bash
+cargo xtask test-ac AC-XK-<AREA>-<NNN>
+```
+
+### Risks
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)